spec/static/main.js

```diff
@@ -217,6 +217,10 @@ app.on('ready', function () {
       window.webContents.send('executeJavaScript-promise-response', result)
     }).catch((error) => {
       window.webContents.send('executeJavaScript-promise-error', error)
+
+      if (error && error.name) {
+        window.webContents.send('executeJavaScript-promise-error-name', error.name)
```

Comment:
Do you need this only for tests? Can the `'executeJavaScript-promise-error'` message be used instead?

Reply:
Sadly not (but this is a wonderful example of how un-intuitive this is) - if we send the received error over the IPC again, we'll get `{}` again 😆